packages/components/src/StatusLabel/StatusLabel.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import classnames from "classnames";
+import styles from "./StatusLabel.css";
+import { Text } from "../Text";
+
+export type StatusType =
+  | "success"
+  | "warning"
+  | "critical"
+  | "inactive"
+  | "informative";
+
+export interface StatusLabelType {
+  readonly statusLabel: string;
+  readonly statusType?: StatusType;
+}
+
+interface StatusLabelProps {
+  /**
+   * Text to display
+   */
+  readonly label: string;
+
+  /**
+   * Alignment of label
```

Comment:
I think something funky is happening with the asterixes being rendered in the documentation
<img width="754" alt="Screen Shot 2022-03-24 at 4 39 12 PM" src="https://user-images.githubusercontent.com/19555235/160021671-bd023d7f-47cd-4014-bee0-743fdbfda6c2.png">

Reply:
Great catch 👁️